libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/RouteViewModel.java

```diff
@@ -303,4 +319,8 @@ private void addNavigationViewOptions(NavigationRoute.Builder builder) {
       .language(locale)
       .voiceUnits(unitType);
   }
+
+  public interface Callback {
```

Comment:
Don't personally like `Callback` name here. Could be confused with `Callback` from OkHttp. It's also kind of vague IMO because it's not giving any information on the context in which is being used.

Reply:
Why are we declaring `Callback` as an inner `interface`? Any reasons in particular (just curious)?